### ISSUE TITLE
Access to the rigid between-frame registrations of the ABCD resting-state, task-based and diffusion data 

### ISSUE THREAD

**Issue by melanieganz** [ABCD-STUDY/nda-abcd-collection-3165#66]
Hi,

I would like to do a simple analysis of the ABCD motion data. I am interested in displacement, but also all rotations and translations and the speed of the motion.
 
Therefore, I would like to get access to the rigid between-frame registrations of the ABCD resting-state, task-based and diffusion data as well as just the age and gender information if that is possible. 
I have permission to access the raw data and can through the NIMH data archive download all the imaging raw-data (all 87 Tb of this data). This means that I could do the registrations myself, but they are already done when running fMRIprep (e.g. I would specifically look for the bold_to_bold_ref_xfm files, etc.).

Also I think the xfm registration matrices plus limited meta data can be considered as anonymous and hence it might be a bit easier to share than identifiable data.
 
Kind regards,
Melanie

**Comment by melanieganz** [comment]
Hi, is there any input you can give to this?

**Comment by ericearl** [comment]
Hey @melanieganz , I don't work in the DCAN Labs anymore, but I do know there should be those numbers in these files:

https://collection3165.readthedocs.io/en/stable/derivatives/#individual-functional-run-and-motion-files-derivativesfuncruns_task-midnbacksstrest

Maybe @mattcieslak can speak to whether QSIPrep puts out the DWI motion data, but at least you can find the fMRI motion in that link above. Furthermore, you can also download ONLY those motion files with the nda-abcd-s3-downloader:

https://github.com/DCAN-Labs/nda-abcd-s3-downloader

Just make sure you provide the data subsets argument and these data subsets:

```
derivatives.func.motion_task-MID
derivatives.func.motion_task-nback
derivatives.func.motion_task-rest
derivatives.func.motion_task-SST
derivatives.func.updated_motion_task-MID
derivatives.func.updated_motion_task-nback
derivatives.func.updated_motion_task-rest
derivatives.func.updated_motion_task-SST
```

Or maybe just the last 4? It's been a long time and I don't remember perfectly...

**Comment by mattcieslak** [comment]
Hi @melanieganz I have head motion from 5999 year1 scans from Eddy, but we'd need to figure out how to share it safely.

**Comment by melanieganz** [comment]
Dear @ericearl and @mattcieslak , thanks so much for your input! We will try the ABCD BIDS downloader @ericearl. And @mattcieslak, I can send you a secure file transfer via the Danish super computing center Deic (https://www.deic.dk/en/filesender). Would that work for you?

**Comment by LuciMoore** [comment]
@melanieganz FYI the [nda-abcd-s3-downloader](https://github.com/DCAN-Labs/nda-abcd-s3-downloader) is currently under construction. We plan on fixing it ASAP, our team is just waiting on our NDA access to be renewed

**Comment by mattcieslak** [comment]
By safely I mean that we'd need to talk with the ABCC PI's and make sure the DUCs are all good. Could we follow up over email?

**Comment by melanieganz** [comment]
> @melanieganz FYI the [nda-abcd-s3-downloader](https://github.com/DCAN-Labs/nda-abcd-s3-downloader) is currently under construction. We plan on fixing it ASAP, our team is just waiting on our NDA access to be renewed

Thanks for the information @LuciMoore!

**Comment by melanieganz** [comment]
> By safely I mean that we'd need to talk with the ABCC PI's and make sure the DUCs are all good. Could we follow up over email?

Yes, I will e-mail you directly and I can send you my NDA credentials. Thank you!

**Comment by LuciMoore** [comment]
@melanieganz I'm going to close this issue as it appears to have been resolved, but feel free to follow up if there are still outstanding issues on your end
